pulp_container/app/migrations/0017_add_granular_perms.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 2.2.17 on 2021-02-03 16:29
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('container', '0016_add_delete_versions_permission'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='containernamespace',
+            options={'permissions': [('namespace_add_containerdistribution', 'Add a distribution to a namespace'), ('namespace_delete_containerdistribution', 'Delete a distribution from a namespace'), ('namespace_view_containerdistribution', 'View a distribution in a namespace'), ('namespace_pull_containerdistribution', 'Pull from all distributions in a namespace'), ('namespace_push_containerdistribution', 'Push to all distributions in a namespace'), ('namespace_change_containerdistribution', 'Change all distributions in a namespace'), ('namespace_view_containerpushrepository', 'View all push repositories in a namespace'), ('namespace_modify_content_containerpushrepository', 'Modify content in all push repositories in a namespace')]},
```

Comment:
this is nitpick, but since we expose this to the user, can we add the distription so it has same pattern not to create confusion?
i.e Delete any distribution from  namespace
Pull from any distribution in a namespace..etc